server/src/main/resources/db/migration/postgresql/V5_0__Additional_Data_Storage_Info.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE object_permissions
+    ADD COLUMN granter_address VARCHAR(44),
+    ADD COLUMN storage_key_address VARCHAR(44),
+    ADD COLUMN is_object_with_meta BOOLEAN
+;
+
+WITH granter_lookup AS (
+    SELECT op.object_hash, dsa.account_address
+    FROM object_permissions op JOIN data_storage_accounts dsa
+    ON op.grantee_address = dsa.account_address
+)
+UPDATE object_permissions op
+SET is_object_with_meta = true,
+    granter_address = granter_lookup.account_address
+FROM granter_lookup
+WHERE granter_lookup.object_hash = op.object_hash;
+
+ALTER TABLE object_permissions
+    ALTER COLUMN granter_address SET NOT NULL,
+    ALTER COLUMN is_object_with_meta SET NOT NULL
+;
+--     ALTER COLUMN storage_key_address SET NOT NULL, -- todo: subsequent migration to make this true after this gets set on startup
```

Comment:
once the bit of code has gone and updated the storage key address for existing records to the master key's address, we can toss another migration in to make this column not nullable